pxfish/paths.py

```diff
@@ -31,7 +31,7 @@ def simplename(name):
     return re.sub(r'\W|^(?=\d)', '_', name).lower()
 
 
-def create_named_path(path, name):
+def create_named_path(path, name, object_type=None):
```

Comment:
would be better to find a different name than `object_type` considering that it is used in Aquarium to mean something else.

in this context, it's a directory name that represents a classification. so maybe `classification` or `subdirectory`